kodilla-testing2/src/test/java/com/kodilla/testing2/crudapp/CruddAppTestSuite.java

```diff
@@ -105,7 +106,33 @@ private boolean checkTaskExistsInTrello(String taskName) throws InterruptedExcep
     @Test
     public void shouldCreateTrelloCard() throws InterruptedException {
```

Comment:
Tu mi się coś mocno nie zgadza. Wywołujesz jeden test z drugiego?
Poza tym ten test `shouldCreateTrelloCard` jest wybrakowany - niczego de facto nie testuje.
Realizacja zadania powinna polegać na tym, że w sekcji `cleanup` testu wywołasz metodę prywatną usuwającą task (nie powinna być oznaczona adnotacją @Test - ona nie jest testem!). Nie ma też potrzeby przypisywać nazwy taska do pola klasy testowej. Jeśli task tworzysz w teście `shouldCreateTrelloCard` - to wystarczy że przekażesz jego nazwę jako argument do metody czyszczącej.